src/test/java/WinningMovieTest.java

```diff
@@ -0,0 +1,39 @@
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.SelenideElement;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.*;
+
+public class WinningMovieTest {
+
+    @BeforeClass
+    public void setUp() {
+        Configuration.browserSize = "1920x1080";
+        Configuration.timeout = 10000; // Таймаут на 10 секунд
+        Configuration.browser = "chrome";
+    }
+
+    @Test
+    public void testSearchForTitanic() {
+
+        // Відкриття IMDb з примусовою зміною мови на англійську
```

Comment:
Usually having comments in the code it not a good practice, code should be self-explanatory.
We use comments only to have a small hint of why something was done, for example, if we run an SQL query in auto test and we exclude 3 customers because they are broken then we will add a comment to explain why, otherwise withing a year people may forget.